bigquery/indexer.py

```diff
@@ -0,0 +1,114 @@
+"""Loads BigQuery table into Elasticsearch."""
+
+import csv
+import json
+import logging
+import os
+import time
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+import pandas as pd
+
+
+# Log to stderr.
+logging.basicConfig(level=logging.INFO,
+    format='%(asctime)s %(filename)10s:%(lineno)s %(levelname)s %(message)s',
+    datefmt='%Y%m%d%H:%M:%S')
+logger = logging.getLogger('indexer.bigquery')
+
+
+def open_and_return_json(file_path):
+  """Opens and returns JSON contents.
+
+  Args:
+    file_path: Relative path of JSON file.
+
+  Returns:
+    Parsed JSON.
+  """
+  with open(file_path,'r') as f:
+    # Remove JSON comments.
+    jsonDict = json.loads('\n'.join([row for row in f.readlines() if len(row.split('//')) == 1]))
```

Comment:
I'm unfamiliar with what bigquery exports look like, but I wasn't aware that JSON supported comments, or is the input not JSON? Looks like we don't expect to see any lines starting with valid JSON and ending with a comment, is that correct?

Reply:
Added clarifying comment